tests/containers/helm.pm

```diff
@@ -42,24 +64,26 @@ sub run {
     assert_script_run("helm list");
 
     # Wait for deployment
-    script_retry("kubectl rollout status deployment/$self->{deployment_name}", delay => 30, retry => 5);
+    script_retry("kubectl rollout status deployment/$self->{deployment_name}", delay => 30, retry => 10);
+    assert_script_run("whereis kubectl");
```

Comment:
What is this for? It's not even the first `kubectl` command.

Reply:
Developing stuff, removed